.circleci/config.yml

```diff
@@ -69,6 +69,7 @@ jobs:
           name: Run unit tests
           command: |
             npm ci
+            npx nuxi prepare
```

Comment:
What's this doing?

Reply:
it generates the .nuxt/tsconfig.json

Reply:
it also generates typescript definition files for the project. 
It's a little weird that it's only mentioned here though: https://v3.nuxtjs.org/migration/configuration#migration-2

Because our root `tsconfig.json` is extending `.nuxt/tsconfig.json`, ([per this setup](https://v3.nuxtjs.org/guide/directory-structure/tsconfig/#typescript-configuration-file))
we need to create it first or vite (test runner) fails when it tries to process the root `tsconfig.json`